test/native/jest.config.js

```diff
@@ -31,11 +31,12 @@ module.exports = {
 	testEnvironment: 'jsdom',
 	testMatch: [
 		'**/test/*.native.[jt]s?(x)',
+		'<rootDir>/packages/react-native-editor/src/**/__tests__/**/*.[jt]s?(x)',
```

Comment:
I left a similar comment in another file, this is fine as a temp solution, but in the long run we should adjust test names to make it seamless for both the web and mobile. I mean, it should be based on conventions so we don't have to tweak config files too much.